lib/Arch/Arch.cpp

```diff
@@ -63,8 +64,6 @@ std::unique_ptr<CallingConvention> CallingConvention::CreateCCFromArch(
     }
 
     // Fallthrough for unsupported architectures
-    case remill::kArchX86_AVX:
-    case remill::kArchX86_AVX512:
     case remill::kArchAMD64_AVX:
```

Comment:
The AVX variants of AMD64 can be moved up to the `case remill::kArchAMD64:` as well.

Reply:
Fixed in 7621822